k1te-backend/src/main/java/ua/com/pragmasoft/k1te/backend/router/domain/Channels.java

```diff
@@ -15,5 +17,15 @@ public interface Channels {
 
   Member find(String channel, String memberId);
 
+  Integer findPinnedMessage(Member from, Member to);
+
+  Member switchConnection(String channelName, String memberId, String newConnection);
+
+  void updatePinnedMessageId(Member from, Member to, Integer pinnedMessagedId);
+
   void updatePeer(Member member, String peerMemberId);
+
+  void updateUri(Member memberToUpdate, String connectionUri, String messageId, Instant usageTime);
```

Comment:
probably updateConnection suits better.. URI is too generic

Reply:
Yeah